frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualResponsePage.tsx

```diff
@@ -189,10 +237,31 @@ export const IndividualResponsePage = (): JSX.Element => {
           </Text>
           <Box display="inline-flex">
             <Text as="span" textStyle="subhead-1">
-              Time:&nbsp;
+              Timestamp:
+            </Text>
+            <Skeleton isLoaded={!isLoading && !isError}>
+              &nbsp;{data?.submissionTime ?? 'Loading...'}
+            </Skeleton>
+          </Box>
+          <Box
+            alignItems="center"
+            display="inline-flex"
+            hidden={attachmentDownloadUrls.size === 0}
+          >
+            <Text as="span" textStyle="subhead-1">
+              Attachments:
             </Text>
             <Skeleton isLoaded={!isLoading && !isError}>
-              {data?.submissionTime ?? 'Loading...'}
+              &nbsp;
+              <Button
+                variant="link"
+                aria-label="Download attachments as ZIP"
+                isDisabled={handleDownloadMutation.isLoading}
```

Comment:
nit: should we use the `isLoading` key instead?

```tsx
isLoading={handleDownloadMutation.isLoading}
```

not sure how the loading spinner will look like though, lol (maybe we can change the right icon to the spinner on loading state if it looks weird)

Reply:
Used for consistency with individual attachment download buttons as well, because the individual attachment download buttons use isDisabled. This is how it looks like for isLoading, I feel like it looks a bit weird? thoughts? haha

<img width="989" alt="image" src="https://user-images.githubusercontent.com/25571626/186659153-fb13de3c-50bf-4187-b300-f1dcccaa62a8.png">

Reply:
tagging @karrui for thoughts, haha

Reply:
Can we change the right icon to spinner when loading then? Can continue using disabled prop over loading.

Reply:
okie, done in [4dfeeab](https://github.com/opengovsg/FormSG/pull/4672/commits/4dfeeab65852da07ee85a782b7495ae883a6d95d)